src/mad_spark_alt/core/simple_qadi_orchestrator.py

```
@@ -467,7 +468,7 @@ def _parse_hypothesis_scores(
             # Check if we've reached the next hypothesis or end section
             if in_section:
                 if re.match(
-                    rf"^(?:-\s*)?H{hypothesis_num + 1}:", line
+                    rf"^(?:-\s*)?(?:\*\*)?H{hypothesis_num + 1}[:.]", line
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)